### ISSUE TITLE
Allow split of signin/signup forms in Passwordless Recipe and ability for custom fields

### ISSUE THREAD

**Issue by jeremylynch** [supertokens/supertokens-auth-react#369]
Under some circumstances, a user may want to collect more than just email address or phone number in the signup process for passwordless (for example, adding a `name` field on signup). Splitting forms and adding custom fields is well documented for [other recipes](https://supertokens.io/docs/emailpassword/common-customizations/signup-form/adding-fields), but is unfortunately not mentioned at all for the Passwordless Recipe. 

Is this possible to achieve?

**Comment by rishabhpoddar** [comment]
@jeremylynch yes. We are working on adding user meta data which will enable this. However, the way we plan on doing this is via a form post sign up instead of while signing up. What do you think?

**Comment by execreate** [comment]
Interested as well, post sign-up sounds fine for me. Are you planning to release an update on this any time soon?
